libraries/AP_BLHeli/AP_BLHeli.h

```diff
@@ -231,6 +231,8 @@ class AP_BLHeli : public AP_ESC_Telem_Backend {
 
     // have we disabled motor outputs?
     bool motors_disabled;
+    // mask of motors that should normally be disabled
+    uint16_t motors_disabled_mask;
```

Comment:
not a big deal of course but we're not really disabling "motors".. we're disabling servo output channels.

Reply:
ok, I can fix

Reply:
we're using motors throughout this file so I think I am just going to leave as-is